db/main/migrate/20180404000001_create_installations.rb

```diff
@@ -0,0 +1,25 @@
+class CreateInstallations < ActiveRecord::Migration
+  PSQL = ActiveRecord::Base.connection.select_value('SELECT version()')
+
+  def self.up
+    create_table :installations do |t|
+      t.belongs_to :owner, :polymorphic => true # will add owner_id and owner_type columns
+      t.integer    :github_id
+      if /PostgreSQL 9.3/.match(PSQL)
+        t.json       :permissions
+      else
+        t.jsonb    :permissions
+      end
+      t.integer    :added_by
+      t.integer    :removed_by
+      t.datetime   :removed_on
+      t.datetime   :updated_on
```

Comment:
This should be replaced with `t.timestamps`, which will automatically create `updated_at` and `created_at`. We might also want to rename `removed_on` to `removed_at` to follow convention.